modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -56,10 +56,10 @@ export default class TileLayer extends CompositeLayer {
           if (!allCurrTilesLoaded) {
             Promise.all(currTiles.map(tile => tile.data)).then(() => {
               this.setState({isLoaded: true});
-              onDataLoaded(currTiles.filter(tile => tile._data).map(tile => tile._data));
+              onTileLoaded(currTiles.filter(tile => tile._data).map(tile => tile._data));
```

Comment:
Reading this again - `onTileLoaded` is not really called when new tile data is loaded, but whenever the viewport is changed. The name is very misleading. Maybe call it `onViewportLoaded`?

Reply:
`onViewportLoaded` is also confusing.